src/mappings/Shelf.ts

```diff
@@ -25,17 +25,17 @@ export function handleShelfIssue(call: IssueCall): void {
   let loanId = loanIdFromPoolIdAndIndex(poolId, loanIndex)
 
   let pool = Pool.load(poolId)
-  let poolChanged = false
 
   if (!pool.loans.includes(poolId)) {
     // TODO: maybe optimize by using a binary search on a sorted array instead
     log.debug('handleShelfIssue: will add loan {} to pool {}', [loanId, poolId])
+    let l1 = pool.loans.length
```

Comment:
If everything works, I think this can be removed again?

Reply:
Done